streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -75,12 +80,7 @@ public void shouldNotAllowZeroTopicsWhenAddingSource() {
 
     @Test(expected = NullPointerException.class)
     public void shouldNotAllowNullNameWhenAddingProcessor() {
-        topology.addProcessor(null, new ProcessorSupplier() {
-            @Override
-            public Processor get() {
-                return new MockProcessorSupplier().get();
-            }
-        });
+        topology.addProcessor(null, () -> new MockProcessorSupplier().get());
```

Comment:
There's a bunch of incidental cleanup in here, just resolving IDEA warnings (many are related to moving to java 8).

Reply:
:)